packages/expo-cli/src/credentials/views/IosProvisioningProfile.ts

```diff
@@ -342,8 +346,28 @@ export async function validateProfileWithoutApple(
     return false;
   }
 
-  if (ProvisioningProfileManager.isExpired(base64EncodedProfile)) {
-    spinner.fail('Provisioning profile on file is expired');
+  const buffer = Buffer.from(base64EncodedProfile, 'base64');
+  const profile = buffer.toString('utf-8');
+  const profilePlist = plist.parse(profile) as PlistObject;
+
+  const distCertFingerprint = await PKCS12Utils.getP12CertFingerprint(
```

Comment:
I think those two functions can throw an exception, probably it's best to extend `try` blockto to the entire function